assignments/assignment02.rb

```diff
@@ -73,3 +111,167 @@ def pluck(ary, key)
 # - daily balance
 # - summary:
 #   - starting balance, total deposits, total withdrawals, ending balance
+
```

Comment:
A few of comments on this.
1. You nailed the html generation via heredocs
2. You missed the daily balances thing (see comment below).
3. You are using iteration with each instead of using map/reduce. Learning how to think about problems so that you can use map/reduce is amazingly powerful.